mmengine/config/config.py

```diff
@@ -1472,8 +1472,7 @@ def _format_dict(input_dict, outest_level=False):
                 blank_line_before_nested_class_or_def=True,
                 split_before_expression_after_opening_paren=True)
             try:
-                text, _ = FormatCode(
-                    text, style_config=yapf_style, verify=True)
+                text, _ = FormatCode(text, style_config=yapf_style)
```

Comment:
Can we add a version checking? If its version is less than 0.40.2, we still pass the `verify=True` to it.